### PR TITLE
Added support for looping

### PR DESCRIPTION
**Pull request by almostintuitive**
I've desperately needed this feature. Please ignore if it doesn't go well with the original concept.
But it made it very useful for me - maybe it'll do the same for others ;).

### CODE REVIEW

**Comment by Haud** [comment]
Hey itchingpixels,

Thanks for contributing! This looks great and is functionality that we can definitely add. I will test this later, but in the meanwhile, can you update the documentation of the `shouldLoopVideo` property to point out that the completion delegate is not called if set to true?

Thank you!

**Comment by almostintuitive** [comment]
Hey,

Done!:)

Cheers!

File: Pod/Classes/PIVideoView.m [hunk 1]
```diff
@@ -149,6 +149,11 @@ - (void)beginPlayingLoadedVideoFile
 {
     NSAssert(self.videoLoaded, @"Call to beginPlayingLoadedVideoFile when no video was loaded.");
     [self registerForAVPlayerNotifications];
+	
+	if (self.shouldLoopVideo) {
+		self.player.actionAtItemEnd = AVPlayerActionAtItemEndNone;
+	}
```
Comment (lk23lekkladf2) [inline Pod/Classes/PIVideoView.m]: - [ ] Fix indentation. It should be aligned at the same level as line https://github.com/prolificinteractive/PIVideoPlayer/pull/2/files#diff-572f697f08361d33ac5f72c846264966R151

File: Pod/Classes/PIVideoView.m [hunk 2]
```diff
@@ -217,9 +222,14 @@ - (void)onVideoLoaded
...
 - (void)videoFinishedPlaying:(id)sender
 {
-    if ([self.delegate respondsToSelector:@selector(videoViewDidFinishPlayingVideo:)]){
-        [self.delegate videoViewDidFinishPlayingVideo:self];
-    }
+	if (self.shouldLoopVideo) {
+		AVPlayerItem *playerItem = [sender object];
+		[playerItem seekToTime:kCMTimeZero];
+	} else {
+		if ([self.delegate respondsToSelector:@selector(videoViewDidFinishPlayingVideo:)]){
```
Comment (lk23lekkladf2) [inline Pod/Classes/PIVideoView.m]: - [ ] Please refactor to use `else if`:

```
} else if { 
     [self.delegate..
```

**Comment by lk23lekkladf2** [comment]
Thanks @itchingpixels for the pull request. I added a couple of style comments to keep things consistent in the project.

**Comment by almostintuitive** [comment]
welcome, they're done (hopefully indentation is ok now)

**Comment by Haud** [comment]
Thanks for the update @itchingpixels . This has been merged and the pod spec has been updated to `0.1.3` on the Cocoapods master spec repo.

**Comment by almostintuitive** [comment]
🍹

**Pull request merged**